django_project/ford3/templates/provider_form.html

```diff
@@ -23,10 +23,17 @@ <h4>Provider Details</h4>
 			</div>
 			<br>
 			<div class="row">
-				<div class="col-4 npl">Provider Logo</div>
+				<div class="col-4 npl">
+                    Provider Logo
+                    <input type="file" name="file_logo">
+                    <button type="submit" class="edu-button edu-button-blue" name="upload_logo">Upload Logo</button>
+
+                </div>
 				<div class="col-8">
 					<div class="provider-form-photo">
-
+                        {% if provider_logo %}
```

Comment:
This is a form template, so I think you can get these variable from `form` model instance that get passed into this template.